docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -367,6 +345,121 @@ In Trino, the same can be achieved with this query:
     FROM collection
     WHERE _id > timestamp_objectid(TIMESTAMP '2021-08-07 17:51:36 +00:00');
 
+.. _mongodb-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and MongoDB each support types that the other does not, this
+connector :ref:`modifies some types <type-mapping-overview>` when reading or
+writing data. Data types may not map the same way in both directions between
+Trino and the data source. Refer to the following sections for type mapping in
+each direction.
+
+MongoDB to Trino read type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps MongoDB types to the corresponding Trino types following
+this table:
+
+.. list-table:: MongoDB to Trino type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - MongoDB type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
```

Comment:
Boolean.

Let's avoid using uppercase because it's not natural for MongoDB users and there's a `ObjectId` type using both upper & lower characters.